sdk/identity/Azure.Identity/src/InteractiveBrowserCredentialOptions.cs

```diff
@@ -34,9 +34,9 @@ public string TenantId
         public string ClientId { get; set; } = Constants.DeveloperSignOnClientId;
 
         /// <summary>
-        /// Specifies the <see cref="TokenCache"/> to be used by the credential.
+        /// Specifies the <see cref="TokenCachePersistenceOptions"/> to be used by the credential.
```

Comment:
This should probably mention if no `TokenCachePersistenceOptions` are specified the cache won't be persisted.